src/content/docs/en/user-guide/bridge.mdx

```diff
@@ -20,32 +20,25 @@ excerpt: "To start bridging assets from Sepolia, navigate to the portal bridge a
 {/* TODO: Update all instructions after being able to walk through the whole flow. */}
 
 
-Visit our [Bridge](https://sepolia.scroll.io/bridge) app to get started![^thanks-hop] The Bridge supports both **Deposit** and **Withdraw** operations, allowing users to trustlessly move assets from Sepolia Testnet to Scroll Sepolia Testnet.
+Visit our bridge app on [Sepolia](https://sepolia.scroll.io/bridge) or [Mainnet](https://scroll.io/bridge) to get started![^thanks-hop] The Bridge supports both **Deposit** and **Withdraw** operations, allowing users to trustlessly move assets from L1 to L2.
```

Comment:
F/U: we can drop hop mention now since the code has heavily diverged from that initial UI.